docs/classified-provisioning-docs/prereqs.md

```diff
@@ -0,0 +1,105 @@
+# Prerequisites
+
+In order to utilize Classified Provisioning in SC2S/C2S, you will need the following prerequisites.
+
+## Tools
+
+The following tools are required to package Classified Provisioning artifacts and install/upgrade Rancher.
+
+- [Hauler](https://rancherfederal.github.io/hauler-docs/docs/introduction/install)
+- [Helm](https://helm.sh/docs/intro/install)
+- [Rancher](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade) (v2.7.13, v2.7.14, v2.8.4 or greater)
```

Comment:
We can do that. To that point, we'd need to amend the `values.yaml` and and the image list to include the whitelabel image (just to be safe that the user has it). That was my biggest concern, that if someone was using upstream and attempted to use our charts, they most likely wouldn't have it and their deployment would fail.